src/FluentValidation.Tests/ValidatorTesterTester.cs

```diff
@@ -86,6 +91,18 @@ public class ValidatorTesterTester {
 			});
 		}
 
+		[Fact]
+		public void ShouldHaveValidationError_Identity_Should_not_throw_when_there_are_validation_errors() {
+			validator.ShouldHaveValidationErrorFor<Person>(x => x, new Person { Forename = "Helle", Surname = "Helle" }, "default");
+		}
+
+		[Fact]
+		public void ShouldHaveValidationError_Identity_Should_throw_when_there_are_no_validation_errors() {

```

Comment:
There are no tests for the new `ShouldNotHaveValidationError` overload, these will need adding please. 

In summary there should a total of 4 new tests:
- ShouldHaveValidationError overload, positive case
- ShouldHaveValidationError negative case (throws an exception which you should be catching and asserting against)
- ShouldNotHaveValidationError overload, positive case
- ShouldNotHaveValidationError negative case (throws an exception which you should be catching and asserting against)  

If you want to be completely thorough then additional tests to cover with/without ruleset would also be welcome, but I'd consider this optional.